tox.ini

```diff
@@ -15,11 +15,10 @@ commands=flake8 dag
 [testenv]
 setenv =
     PYTHONPATH = {toxinidir}
-deps =
-    -r{toxinidir}/requirements_dev.txt
+
+deps=.
 commands =
-    pip install -U pip
-    py.test --basetemp={envtmpdir}
+    python {toxinidir}/setup.py test
```

Comment:
Can you please revert this change, I prefer using pytest and this is important on CI servers to install the latest pip and then run the test command

Reply:
But doesn't `tox` create a virtualenv for each test environment? With  `pip` installed by default?
At least `virtualenv` does that by default unless you forbid it. 

When it comes to calling `pytest` directly, I thought it would make sense to make use of the `test` target alias set in `setup.cfg` (hence `python {toxinidir}/setup.py test`) so that if we decide to switch test frameworks for whatever reason, tox would still work and there would be one file less to visit for updating the build.